atlasdb-conjure/src/main/java/com/palantir/atlasdb/http/v2/ClientOptions.java

```diff
@@ -36,8 +36,8 @@
     // TODO (jkong): Re-enable client QoS after response body leaks are handled correctly.
     // Throws after expected outages of 1/2 * 0.01 * (2^11 - 1) = 10.24 s
     public static final ClientOptions DEFAULT_RETRYING = ImmutableClientOptions.builder()
-            .connectTimeout(Duration.ofSeconds(10))
-            .readTimeout(Duration.ofSeconds(65))
+            .connectTimeout(Duration.ofSeconds(1))
```

Comment:
This is as per https://github.com/palantir/atlasdb/issues/4412; I would probably have an even lower timeout, most timelock calls take a few single digit milliseconds.

Reply:
This is fine, yeah I'd have a lower timeout.